File: libra-view-modules/movie/src/main/java/hu/bozgab/movie/dto/integration/movies/TMDBSearchMovieResponse.java
package hu.bozgab.movie.dto.integration.movies;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@Data
@NoArgsConstructor
@AllArgsConstructor
public class TMDBSearchMovieResponse {

    private int page;

    private List<TMDBMovieDetailsDTO> results;

    private int total_pages;

    private int total_results;

}

File: libra-view-web/src/app/shared/data-model/cinematic/integration/search/tmdb-search-movie.response.ts
import { TMDBMovieGeneral } from '../movies/tmdb-movie-general'
import { TMDBSearchResponse } from './tmdb-search.response'

export interface TMDBSearchMovieResponse extends TMDBSearchResponse {
  discriminator: 'MOVIE'

  page: number

  results: Array<TMDBMovieGeneral>

  total_pages: number

  total_results: number
}
